index.html

```diff
@@ -292,6 +293,7 @@ <h1>I'm always interested in hearing about new projects, so if you'd like to cha
       <a href="#"> <img src="images/icons/medium.png" alt=""> </a>
     </li>
    </ul>
```

Comment:
- [ ] Kindly move your social links a little to the right as shown in the template

| Your Design | Figma Design |
|-------------|---------------|
|||